app/schema/api.py

```diff
@@ -105,20 +105,23 @@ class BaseMakeReservationIn(BaseModel):
     """BaseMakeReservationIn."""
 
     proj_id: UUID
+    group_id: UUID | None = None
     type: ServiceType
     subtype: ServiceSubtype
 
 
 class MakeOneshotReservationIn(BaseMakeReservationIn):
     """MakeOneshotReservationIn."""
 
+    user_id: UUID
```

Comment:
user_id can be moved from MakeOneshotReservationIn and MakeLongrunReservationIn to BaseMakeReservationIn.
This will fix also the linting error (actions were still disabled in this repo, but now they are running)

Reply:
Should be done now.